### PR TITLE
feat: mint-css package

### PR DESCRIPTION
**Pull request by shubhamgavali1998**
## What does this PR do?
A CSS library that provides classes, tokens, variables, fonts and other essential stylings.

## What packages have been affected by this PR?
N.A.

## Types of changes

What types of changes does your code introduce to this project?

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)

- [x] New feature (non-breaking change which adds functionality)

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Package version increase in any of the packages?
0.0.1

## Checklist before merging

- [x] These changes have been thoroughly tested.

- [x] Changes need to be immediately published on npm.

### CODE REVIEW

**Review by subhamkapiswe — APPROVED** [review]

**Review by AkshayNaikGroww — APPROVED** [review]

**Review by AkshayNaikGroww — APPROVED** [review]

File: packages/mint-css/typography/fonts/font-face.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
+@font-face {
+  font-family: 'GrowwSans';
+  font-style: normal;
+  font-weight: 300 700;
+  src: url(./GrowwSans-Variable.woff2) format('woff2-variations');
+}
+
+@font-face {
```
Comment (AkshayNaikGroww) [inline packages/mint-css/typography/fonts/font-face.css:8]: @prashant-adhikari-groww lets send NotoSans in the first version itself ?

**Review by AkshayNaikGroww — APPROVED** [review]

File: packages/mint-css/typography/fonts/font-face.css (hunk 1 above)
Comment (prashant-adhikari-groww) [reply packages/mint-css/typography/fonts/font-face.css:8]: HNS will be affected due to this. Currently app haven't made it live

Comment (AkshayNaikGroww) [reply packages/mint-css/typography/fonts/font-face.css:8]: @shubhamgavali1998 Lets pull back NotoSans from this version, we'll send it in later releases/version

File: packages/mint-css/theme/variables/index.css [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+  --green100: #EBF9F5;
+  --purple500: #5367ff;
+  --purple300: #84a4ff;
+  --purple100: #eef0ff;
+  --yellow500: #ffb61b;
+  --yellow100: #fff5e0;
+  --red500: #eb5b3c;
+  --red100: #fae9e5;
+  --tempBg: #ffffff;
+  --tempTransparent:  rgba(0, 0, 0, 0);
```
Comment (prashant-adhikari-groww) [inline packages/mint-css/theme/variables/index.css:35]: can add comment here why this is used

File: packages/mint-css/typography/fonts/NotoSansDevanagari-Regular.woff2
Comment (prashant-adhikari-groww) [inline packages/mint-css/typography/fonts/NotoSansDevanagari-Regular.woff2:1]: NOTO sans we can send afterwards

File: packages/mint-css/typography/variables/index.css [hunk 3]
```diff
@@ -0,0 +1,20 @@
+html {
+  /* Font sizes */
+  --font-size-12: .75rem;
```
Comment (prashant-adhikari-groww) [inline packages/mint-css/typography/variables/index.css:3]: 0.75rem

File: packages/mint-css/package.json [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  },
+  "keywords": [
+    "mint-css",
+    "css",
+    "groww",
+    "style"
+  ],
+  "contributors": [
+    {
+      "name": "Shubham Gavali"
```
Comment (prashant-adhikari-groww) [inline packages/mint-css/package.json:25]: Add your linkedin profile also

File: packages/mint-css/package.json [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    },
+    {
+      "name": "Akshay Naik"
+    }
+  ],
+  "license": "MIT",
+  "devDependencies": {
+    "postcss": "^8.4.21",
```
Comment (prashant-adhikari-groww) [inline packages/mint-css/package.json:33]: Remove ^

File: packages/mint-css/package.json (hunk 4 above)
Comment (prashant-adhikari-groww) [reply packages/mint-css/package.json:25]: Mera naam bhi daldo guysss 🥲

File: packages/mint-css/typography/fonts/font-face.css (hunk 1 above)
Comment (shubhamgavali70) [reply packages/mint-css/typography/fonts/font-face.css]: Removed

Comment (shubhamgavali1998) [reply packages/mint-css/typography/fonts/font-face.css]: Removed

File: packages/mint-css/theme/variables/index.css (hunk 2 above)
Comment (AkshayNaikGroww) [reply packages/mint-css/theme/variables/index.css:35]: @shubhamgavali1998 I missed adding the comment here, The same comment line is to be added, as I added in scripts.

Comment (shubhamgavali1998) [reply packages/mint-css/theme/variables/index.css:36]: Done

File: packages/mint-css/base/utility.css [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+  width: 100%;
+}
+
+.card {
+  border-radius: 5px;
+  box-shadow: 0 1px 5px 0 var(--boxShadow);
+  border: 1px solid var(--border);
+}
+
+.clickable-text {
```
Comment (shivesh-raj-nigam) [inline packages/mint-css/base/utility.css:132]: check for tokens

File: packages/mint-css/base/utility.css [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+}
+
+.blurEffect1 {
+  filter: blur(3px);
+}
+
+/* clearfix when parent contains float childrens */
+
+.clearfix::after {
+  content: ".";
```
Comment (shivesh-raj-nigam) [inline packages/mint-css/base/utility.css:161]: IG it should be content: ""

**Review by prashant-adhikari-groww — APPROVED** [review]

File: packages/mint-css/base/app.css [hunk 8]
```diff
@@ -0,0 +1,85 @@
+html {
+  box-sizing: border-box;
```
Comment (prashant-adhikari-groww) [inline packages/mint-css/base/app.css:2]: Add font-size 16px.
Verify with Vedant once

File: packages/mint-css/base/utility.css (hunk 6 above)
Comment (shubhamgavali1998) [reply packages/mint-css/base/utility.css:132]: This PR has only package changes, we have another PR which has migration changes, which we will be merging today only after this PR

**Review by ghost — APPROVED** [review]

**Pull request merged**